voluncheer/opportunityboard/views/postanopportunity.py

```diff
@@ -26,7 +26,11 @@ def post_an_opportunity(request):
         if form.is_valid():
             form.save()
         else:
-            print(form.errors.as_data())
```

Comment:
If the request is a post and form is invalid, then it shouldn't render.

Reply:
See above